docs/index.rst

```diff
@@ -1,55 +1,49 @@
-.. _containers-home:
+:product: Container Connectors
+:product: BIG-IP Controller for Kubernetes
+:product: BIG-IP Controller for Cloud Foundry
+:product: BIG-IP Controller for Marathon
+:type: concept
 
-.. index::
-   single: F5 Container Integration; overview
+.. _containers-home:
 
+Introduction to F5 Container Connectors
+=======================================
 
-Introduction to F5 Container Integrations
-=========================================
+F5's Container Connectors (CCs) make it possible to dynamically allocate BIG-IP L4-L7 services in container orchestration environments. Each Container Connector understands the container orchestration environment ('COE'), thereby providing a PaaS-native integration for F5 BIG-IP devices.
 
-F5's Container Integrations make it possible to dynamically allocate BIG-IP L4-L7 services in container orchestration environments. The Container Connectors ('CCs') understand the container orchestration environment ('COE'), thereby providing PaaS-native integrations for F5 BIG-IP devices.
+.. table:: F5 Container Connectors
 
-=======================     ===================================================
-Container Connector         Description
-=======================     ===================================================
-`cf-bigip-ctlr`_            Integrates BIG-IP as an
-                            Application Delivery Controller (ADC) in
-                            `Pivotal Cloud Foundry`_, serving North-South
-                            traffic.
------------------------     ---------------------------------------------------
-`k8s-bigip-ctlr`_           Configures a BIG-IP device to expose applications
-                            in `Kubernetes`_ and `Red Hat OpenShift`_ clusters
-                            as virtual servers, serving North-South traffic.
------------------------     ---------------------------------------------------
-`marathon-bigip-ctlr`_      Configures a BIG-IP device to expose applications
-                            in an `Apache Mesos`_ `Marathon`_ cluster as
-                            virtual servers, serving North-South traffic.
-=======================     ===================================================
+   =======================     ===================================================
+   Container Connector         Description
+   =======================     ===================================================
+   `cf-bigip-ctlr`_            Integrates BIG-IP as an
```

Comment:
Should this description be different than the following descriptions? It does the same thing as the two others so I don't want to give the impression it's somehow different.